assets/src/design-system/components/contextMenu/menuItem.js

```diff
@@ -90,12 +98,12 @@ export const MenuItem = ({
         >
           {label}
         </ItemText>
-        {shortcut && (
+        {shortcut?.display && (
           <Shortcut
             size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
-            forwardedAs="span"
+            forwardedAs="kbd"
           >
-            {shortcut}
+            {shortcut?.display}
           </Shortcut>
         )}
```

Comment:
💭 Should this only be rendered if it's not an icon menu?

Reply:
`Icon` returns early above t his so i think we're ok.